src/components/rating/Star.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Icon, {types, colors} from '../icons/Icon';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+const Star = ({size, checked, onClick = () => undefined}) => {
```

Comment:
no no no :) you misses my point if there is no onClick function we don't add it as property